prisma/schema.prisma

```diff
@@ -50,14 +50,15 @@ model Session {
 }
 
 model User {
-    id            String         @id @default(cuid())
-    name          String?
-    email         String?        @unique
-    emailVerified DateTime?
-    image         String?
-    accounts      Account[]
-    sessions      Session[]
-    OpenAIModels  OpenAIModels[]
+    id                String         @id @default(cuid())
+    name              String?
+    email             String?        @unique
+    emailVerified     DateTime?
+    image             String?
+    interfaceLanguage String?
+    accounts          Account[]
+    sessions          Session[]
+    OpenAIModels      OpenAIModels[]
```

Comment:
remove this, it is unnecessary complexity

Reply:
Removed OpenAiModels column from User model and availableForImageGeneration column from OpenAIModels.